src/systemTest/java/com/flightstats/hub/system/functional/ChannelLatestUpdatedStateTest.java

```diff
@@ -0,0 +1,161 @@
+package com.flightstats.hub.system.functional;
+
+import com.flightstats.hub.model.ChannelConfig;
+import com.flightstats.hub.model.ChannelItemWithBody;
+import com.flightstats.hub.system.extension.TestSuiteClassWrapper;
+import com.flightstats.hub.system.service.ChannelConfigService;
+import com.flightstats.hub.system.service.ChannelItemCreator;
+import com.flightstats.hub.system.service.ChannelItemRetriever;
+import com.flightstats.hub.system.service.InternalZookeeperService;
+import org.joda.time.DateTime;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import javax.inject.Inject;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.SortedSet;
+import java.util.concurrent.TimeUnit;
+
+import static com.flightstats.hub.util.StringUtils.randomAlphaNumeric;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+class ChannelLatestUpdatedStateTest extends TestSuiteClassWrapper {
+    private static final String NO_CHANNEL_LATEST_ZK_VALUE = "1970/01/01/00/00/00/001/none";
+    private String channelName;
+
+    @Inject
+    private ChannelConfigService channelConfigService;
+    @Inject
+    private ChannelItemRetriever channelItemRetriever;
+    @Inject
+    private ChannelItemCreator channelItemCreator;
+    @Inject
+    private InternalZookeeperService zookeeperService;
+
+    @BeforeEach
+    void setup() {
+        channelName = randomAlphaNumeric(10);
+    }
+
+    @AfterEach
+    void cleanup() {
+        channelConfigService.delete(channelName);
+    }
+
+    @Test
+    void channelWithNoItem_hasStateValueOfEpochNone_and_hasNoChannelLatest() {
+        channelConfigService.createWithDefaults(channelName);
```

Comment:
I think this and line 60 can be combined into the before each setup

Reply:
I had it that way originally, but some tests use an historical channel and some don't and it was annoying to have to create a second, different channel with a different channel name. this way it's a bit more explicit.